drgn/helpers/linux/mm.py

```diff
@@ -14,8 +14,9 @@
 from typing import Iterator, List, Optional, Union, overload
 
 from _drgn import _linux_helper_read_vm
-from drgn import IntegerLike, Object, Program, cast
+from drgn import FaultError, IntegerLike, Object, Program, Type, cast
```

Comment:
I don't think you're using either one of these imports now that you limited it to the `slab_cache` helpers.